docs/pages/core-concepts.mdx

```diff
@@ -132,15 +132,15 @@ on GitHub.
 You can find a detailed comparison of the features available in each Teleport 
 edition in [Choose an edition](index.mdx).
 
-### Teleport Enterprise Cloud
+### Teleport Enterprise
```

Comment:
Are we going to have two "Teleport Enterprise" sections in the Core Concepts page after this change? We can use "Teleport Enterprise (Cloud)" and "Teleport Enterprise (Self-Hosted)" instead

Reply:
Changed this to:

### Teleport Enterprise (Cloud)
and
### Teleport Enterprise (Self-Hosted)